ansible/roles/stig/tasks/TOSS_04_010350.yml

```
@@ -25,6 +25,7 @@
           - uidzero.stdout_lines | length == 0
         fail_msg: "Another account has been found to have uid 0 in /etc/passwd besides root."
         quiet: true
+  ignore_errors: true
```

Why ignore errors here?

---

This falls under the explanation above: when this "assert" module activates, it stops the playbook from completing. This will at least have the task show the message this task failed and move on, but will show up as "skipped" in the recap.